lib/themes/baseTokens/seekAsia.ts

```diff
@@ -201,11 +200,11 @@ export const makeTokens = ({
       background: {
         brand,
         input: white,
-        inputDisabled: grey4,
+        inputDisabled: grey5,
         brandAccent,
         formAccent,
         formAccentDisabled: grey4,
-        selection,
+        selection: grey5,
```

Comment:
This changes the highlight colour when selecting suggestions in `Autosuggest`. Can you double check that this is what you want?

Reply:
You're right. I'll revoke that colour change